### PR TITLE
Add suffixes to typos from "ini" to "it"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -28900,12 +28900,28 @@ iniate->initiate
 iniated->initiated
 iniates->initiates
 iniating->initiating
+iniation->initiation
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:28903]: Any `iniation->indication` (and friends) typo?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -29060,6 +29094,8 @@ initates->initiates, imitates,
 initating->initiating, imitating,
 initation->initiation, imitation,
 initations->initiations, imitations,
+initative->initiative, imitative,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29097]: Again, no typo of `indicative`? Perhaps even `limitative`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -29503,8 +29541,12 @@ innitials->initials
 innocous->innocuous
 innoculate->inoculate
 innoculated->inoculated
+innosence->innocence
 innosense->innocence
+innosent->innocent
 inocence->innocence
+inocent->innocent
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29548]: Never a typo of `incent`?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -29578,9 +29620,20 @@ inro->into
 ins't->isn't
 insall->install
 insallation->installation
+insallations->installations
 insalled->installed
+insaller->installer
+insallers->installers
 insalling->installing
+insalls->installs
 insance->instance, insane,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29629]: Perhaps `insance->insurance` (and plural) too?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -29711,7 +29772,14 @@ instatiates->instantiates
 instatiating->instantiating
 instatiation->instantiation
 instatiations->instantiations
+insteall->install, instead,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29775]: Any `insteall->instill` (and friends) misspellings?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -29769,6 +29845,8 @@ instrall->install
 instralled->installed
 instralling->installing
 instralls->installs
+instrance->instance, insurance,
+instrances->instances
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:29849]: Shouldn't plural be consistent with singular?
```suggestion
instrances->instances
instrances->instances, insurances,
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -30005,7 +30129,14 @@ interaxtions->interactions
...
 intercahnge->interchange
 intercahnged->interchanged
+intercat->interact
+intercated->interacted
+intercating->interacting
 intercation->interaction
+intercations->interactions
+intercative->interactive
+intercatively->interactively
+intercats->interacts
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:30139]: Perhaps `inter-caste`?

**Review by DimitriPapadopoulos — COMMENTED** [review]
Some far-fetched suggestions, feel free to ignore them.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:28903]: I couldn't find these typos.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:29097]: I couldn't find these typos.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:29548]: No

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:29775]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:30139]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**